back/engines/commercial/content_builder/app/policies/content_builder/layout_image_policy.rb

```diff
@@ -0,0 +1,18 @@
+module ContentBuilder
+  class LayoutImagePolicy < ApplicationPolicy
+    def create?
+      # The content buildable is not provided and
+      # so we cannot check if the user moderates
+      # the corresponding context. Instead, we
+      # check if the user has any moderation
+      # rights.
+      user&.active? && user_role_service.moderates_something?(user)
```

Comment:
I have seen this pattern of checking for `active?` before sending a message to a service elsewhere in the codebase and I am wondering whether `moderates_something?` shouldn't do `user&.active?` instead of all the callers.

Reply:
But it also annoys me a bit to see `active?` everywhere throughout the different policies. Perhaps we could instead consider it as a tech debt task?

Reply:
I agree, although we should not move stuff to tech debt tasks when we can handle them as part of a task or sprint. In this case, I am not sure. How big would the refactoring be? Let's discuss in a PP session and create a new PR when we come to the conclusion that we can handle it now instead of in a tech debt task in the future.